keras/layers/activations/hard_swish.py

```diff
@@ -0,0 +1,37 @@
+from keras import activations
+from keras.api_export import keras_export
+from keras.layers.layer import Layer
+
+
+@keras_export("keras.layers.HardSwish")
```

Comment:
Given that the activation does not have class-level arguments or state, it does not need to be a layer. It can just be used as a layer via `Activation("hard_swish")`. You can remove the `layer/` changes.

Reply:
Done.